src/status_im/utils/universal_links/core.cljs

```diff
@@ -42,13 +42,13 @@
 
 (fx/defn handle-public-chat [cofx public-chat]
   (log/info "universal-links: handling public chat " public-chat)
-  (chat.events/create-new-public-chat cofx public-chat false))
+  (chat/start-public-chat cofx public-chat false))
 
 (fx/defn handle-view-profile [{:keys [db] :as cofx} profile-id]
   (log/info "universal-links: handling view profile" profile-id)
   (if (new-chat.db/own-whisper-identity? db profile-id)
     (navigation/navigate-to-cofx cofx :my-profile nil)
-    (chat.events/show-profile cofx profile-id)))
+    (navigation/navigate-to-cofx (assoc-in cofx [:db :contacts/identity] profile-id) :profile nil)))
```

Comment:
would be cleaner to do that through fx/merge

Reply:
actually it's the same thing here, should be navigate-to-profile fn

Reply:
Will be done when the profile module is refactored into agreed pattern.